### PR TITLE
Read the code of Bbcode and replace by html tags in user-page-loader.js

### PR DESCRIPTION
**Pull request by vado21**

### CODE REVIEW

**Review by kassidywebb — APPROVED** [review]

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -35,66 +35,66 @@ const parameterUsername = urlParams.get('user');
...
-             sendOption.classList.remove('hidden');
-          } else {
-             document.getElementById('private-option-checkbox').checked = true;
-          }
-           document.getElementById('about-me-form').classList.remove('hidden');
-         }
-       });
-
- }
+function showMessageFormIfLoggedIn() {
```
Comment (ryanluo) [inline src/main/webapp/js/user-page-loader.js:51]: Let's avoid formatting other people's code unless you've made a change.

**Review by ryanluo — APPROVED** [review]
Looks great!

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (vado21) [reply src/main/webapp/js/user-page-loader.js:51]: Ok, my error

**Comment by ryanluo** [comment]
Go ahead and merge this please.

**Pull request merged**